spec/models/payment_receipt_spec.rb

```diff
@@ -7,4 +7,23 @@
 
   it { is_expected.to belong_to(:payment) }
   it { is_expected.to belong_to(:organization) }
+  it { is_expected.to have_one_attached(:file) }
```

Comment:
I think we should start saving on `it` since our test suite is getting pretty long.
Obviously here it's not an issue with 3 its but I'm saying this as a general rule, we should try to refactor when we can.
See https://github.com/getlago/lago-api/pull/3167
```suggestion
  it do
   expect(subject).to ...
   ...
 end
```